src/BankingApp.Data/CustomerDbContext.cs

```diff
@@ -0,0 +1,19 @@
+﻿using BankingApp.Data.Entities;
+using Microsoft.EntityFrameworkCore;
+
+
+namespace BankingApp.Data
+{
+    public class CustomerDbContext : DbContext
+    {
+        public CustomerDbContext(DbContextOptions<CustomerDbContext> options)
+            : base(options)
+        {
+        }
+
+        public DbSet<Customer> Customers { get; set; }
```

Comment:
remove this line of code , the class hasn't been created yet and its out of scope.